packages/commerce-sdk-react/src/hooks/useLocalStorage.ts

```diff
@@ -22,22 +22,25 @@ function useLocalStorage(key: string): Value {
         return window.localStorage.getItem(key)
     }
 
-    const [storedValue, setStoredValue] = useState<Value>(readValue)
+    const useLocalStorageSubscribe = (callback: any) => {
```

Comment:
Consider renaming this function to be something else. Right now it's a function masquerading as a hook, at least with the `use` prefix.

Reply:
Another observation is that you are defining these functions everytime this hook re-renders.. you might want to look into using the `useCallback` hook to cache the function implementations between renders to ensure we are being as performant as possible.

Reply:
Alternatively you can define those functions in a separate file, or outside of the hook implementation.